lib/rbs/test.rb

```diff
@@ -23,6 +24,8 @@ module Test
     METHODS = Kernel.instance_method(:methods)
 
     class ArgumentsReturn
+      include Guaranteed::Inspect
```

Comment:
Without this, you can't call `.inspect` on `ArgumentsReturn` of test types

Reply:
```suggestion
      include _Inspect
```
(yes you can do this)